llvm/lib/MC/ELFObjectWriter.cpp

```diff
@@ -1008,6 +1011,8 @@ void ELFWriter::writeSectionHeaders(const MCAssembler &Asm) {
 
     writeSectionHeader(GroupSymbolIndex, Offsets.first, Size, *Section);
   }
+
+  stats::SectionHeaderBytes += W.OS.tell() - Start;
```

Comment:
Add a test?

If llvm-project ever gets https://maskray.me/blog/2024-03-31-a-compact-section-header-table-for-elf (impl: 
https://github.com/MaskRay/llvm-project/tree/demo-cshdr ; which might need certain extent of agreement), this can be used to show the size difference due to -Wa,--cshdr  :)

Reply:
I don't really want to hardcode the value of this stat in the test, and I'm not sure how else I'd test it?

Reply:
done